website/src/content/api-reference/commercetools-frontend-application-shell.mdx

```diff
@@ -312,6 +315,7 @@ The `THttpClientConfig` object accepts the following options:
   * `uri` (**required**): The URL of the external API to forward the request to.
   * `headers` (optional): Additional HTTP headers to be included in the request to the external API.
   * `audiencePolicy` (optional): See [configuring the audience policy](/concepts/integrate-with-your-own-api#configuring-the-audience-policy).
+  * `includeUserPermissions` (optional): Whether to include an additional token claim in forwarded request containing the logged-in user permissions. See [configuring custom claims](/concepts/integrate-with-your-own-api#configuring-custom-claims)
```

Comment:
I would simply point to the other docs section, where all the necessary info are defined. No need to duplicate part of it.

```suggestion
  * `includeUserPermissions` (optional): See [configuring custom claims](/concepts/integrate-with-your-own-api#configuring-custom-claims).
```

Reply:
Updated in this commit: 088fb79b95c5c8d8d309617db4f8f9f1a47da13b